### PR TITLE
Separate About and Partner sections into two different pages

### PR DESCRIPTION
**Pull request by YusufBolden**
Ran into a slight hiccup with MyQuer which Elle was able to guide me through. Pushed earlier but realized I didn't pull first, so I deleted that requests to save time from working through merge conflicts. There is some remaining whitespace at the bottom of the About page and we'll have to determine what to do with it. Also, the Partners page could use some additional content.

### CODE REVIEW

**Comment by YusufBolden** [comment]
Spent several hours breaking the pages and fixing them to familiarize myself with the process

File: src/pages/partners.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  const sponsors = data.sponsors.nodes;
+
+  // const getImgSrc = name =>
+  //   teamPics.filter(image => {
+  //     if (!image.name.includes(name)) {
+  //       return null
+  //     }
+
+  //     return image
+  //   })[0].childImageSharp.fluid
```
Comment (elleneub) [inline src/pages/partners.js:39]: Want to remove this stuff about the team? Doesn't seem like we need it here.

File: src/pages/partners.module.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ * Partners
+ *
+ */
+
+.team {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(168px, 168px));
+  column-gap: var(--sp-max);
+  row-gap: var(--sp-loose);
+}
```
Comment (elleneub) [inline src/pages/partners.module.scss:13]: It'd be good to go through this and remove any styles that aren't being used in this section. We can talk if you'd like help figuring that out, the way it's happening in the project is kinda odd and took me quite a while to figure out.

**Review by elleneub — CHANGES_REQUESTED** [review]
Looking really good! Couple things to work on but all in all quite good. the about.module.scss could probably have some styles deleted out of it as well now that it doesn't have the partners section

**Review by elleneub — APPROVED** [review]
Looks great, well done!

**Pull request merged**